### DISCUSSION TITLE
如果有OAuth2.0票据泄露的可以参考此路径删除授权

### DISCUSSION THREAD

**Discussion by FDrag0n** [XiaoMi/ha_xiaomi_home#211] [General]
因为之前参与治理过公司的密钥泄露问题，所以习惯性的找了一下注销已有票据的路径

具体路径如下：
米家APP -> 我的 -> 点上面的账号管理进入小米账号页面 -> 应用授权 -> 找到(Xiaomi Home (Home Assistant Integration)) -> 取消授权 即可

**Comment by JimmyKmi** [comment]
感谢 @FDrag0n  提供方法，已增加至wiki：https://note.p.lzj.ac.cn/s/bd635048-ae38-43c9-9f31-d61175a01cf6/doc/5yw25a6d5bi46keb6zeu6aky-blJLGnBrac
